test/utils/Processing.test.ts

```diff
@@ -426,6 +426,12 @@ describe('Processing', () => {
       expect(config.fhirVersion).toEqual(['4.5.0']);
     });
 
+    it('should allow FHIR R6', () => {
+      const input = path.join(__dirname, 'fixtures', 'fhir-r6');
+      const config = readConfig(input);
+      expect(config.fhirVersion).toEqual(['6.0.0-ballot2']);
+    });
```

Comment:
I don't know what the difference is, but I was also able to use just `6.0.0` as a `fhirVersion` already. Is it worth adding a test for that? SUSHI recognizes it as a non-preRelease version, thanks to the snippet of code Chris linked.

Reply:
Getting a package downloaded for `6.0.0` right now is not so good, but either way, I can add a test for when that version is specified.